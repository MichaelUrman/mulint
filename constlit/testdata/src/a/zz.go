// Package a also tests simple literal replacements with a renamed import
package a

import zz "image/png"

var zznah = -2

var zzbs zz.CompressionLevel = -2 // want "Replace `-2` with `zz.BestSpeed`"

func zzCompress(level zz.CompressionLevel) {}

func zzmain() {
	zzCompress(-3) // want "Replace `-3` with `zz.BestCompression`"
}
