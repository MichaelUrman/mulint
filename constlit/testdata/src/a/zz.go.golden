// Package a also tests simple literal replacements with a renamed import
package a

import zz "image/png"

var zznah = -2

var zzbs zz.CompressionLevel = zz.BestSpeed // want "Replace `-2` with `zz.BestSpeed`"

func zzCompress(level zz.CompressionLevel) {}

func zzmain() {
	zzCompress(zz.BestCompression) // want "Replace `-3` with `zz.BestCompression`"
}
